speed.cpp

```diff
@@ -0,0 +1,23 @@
+#include "speed.h"
+
+Speed::Speed() : speed(25) {}
+
+void Speed::setup(Ui::MainWindow *ui) {
+  this->ui = ui;
+  this->speed = 25;
+  this->ui->speed->display(speed);
+}
+
+void Speed::increaseSpeed(int value) {
+  if (speed + value < 99) {
+    this->speed += value;
+    this->ui->speed->display(speed);
+  }
+}
+
+void Speed::decreaseSpeed(int value) {
```

Comment:
If the car has the ability to back up, maybe there should be an indication of that its going backwards by showing negative speed. In other words, we shouldn't have a lower limit for speed.

Reply:
We will need to implement a gear system for that, specifying if we are in drive or in reverse. That is another label we will be need.